www/spline.js

```diff
@@ -21,44 +25,60 @@ function point(x, y, speed) {
         set: function(value){ _speed = value }
     });
 }
-
+/*
```

Comment:
We should get in a habit of writing doxygen style comments for all of our functions

If you type /** and press enter webstorm will generate one for you, then you just fill in the info

So for this function the comment would look like:

/**
 * Function takes two waypoints and returns generated points in a spline between the two given waypoints
 * @param w1 Start waypoint
 * @param w2 End waypoint
 * @constructor Constructs spline object
 */